apps/nextjs-website/src/lib/homepage.ts

```diff
@@ -46,6 +61,20 @@ export const makeHomepageProps = (
   ...makeHomepagePropsFromStatic(staticHeader, staticHomepage),
   comingsoonDocumentation:
     strapiHomepage.data.attributes.comingsoonDocumentation,
+  productsShowcase: {
+    title: strapiHomepage.data.attributes.productsShowcase.title,
+    products: strapiHomepage.data.attributes.productsShowcase.products.data.map(
+      (product) => ({
+        name: product.attributes.name,
+        description: product.attributes.description,
+        slug: product.attributes.slug,
+        logo: {
+          ...product.attributes.logo.data.attributes,
+          url: `${product.attributes.logo.data.attributes.url}`,
+        },
```

Comment:
Also this fallback is needed.
```suggestion
        // TODO: remove undefined fallback when Strapi will manage media in localhost properly
        logo: product.attributes.logo.data
          ? {
              ...product.attributes.logo.data.attributes,
            }
          : {
              url: '',
            },
```